src/accumulator.jl

```diff
@@ -41,6 +41,8 @@ values(ct::Accumulator) = values(ct.map)
 
 sum(ct::Accumulator) = sum(values(ct.map))
 
+most_common(ct::Accumulator) = sort(collect(ct.map), by=p->p[2], rev=true)
```

Comment:
http://stackoverflow.com/questions/5380568/algorithm-to-find-k-smallest-numbers-in-array-of-n-items looks like it may have some implementations that are `O(n)` but I haven't looked at them too carefully to verify correctness.